packages/inventory-compliance/src/index.js

```diff
@@ -1 +1,2 @@
 export { default as default } from './Compliance';
+export { default as SystemRulesTable } from './SystemRulesTable';
```

Comment:
@akofink the line above was `export { Compliance as default } ` before I changed it, which is what caused the problem you saw. Since compliance-frontend uses the default export, it was failing to pick up any as it was undefined.